packages/types/src/legacy/reference-config.ts

```diff
@@ -175,6 +175,62 @@ export type ReferenceConfiguration = {
    * @default undefined
    */
   pathRouting?: PathRouting
+  /**
```

Comment:
these good @hanspagel ?

Reply:
`customModelId` (and the others) sound like you could pass a specific id (as a string or something).

Maybe we use a `generate*` prefix to indicate we expect a callback. And I’m used to refer to those route segments as slugs. So here’s an alternative suggestion:

`generateModelSlug`

Wdyt? Same applies to the others of course.

Reply:
Love it! I figured we would have to rename em lol

Reply:
maybe overrideModelSlug? Since we have a generation function already? Or does having the same name imply override

Reply:
I’d prefer generate, but I’m okay with both!